providers/os/resources/os.lr

```diff
@@ -659,6 +659,9 @@ package @defaults("name version") {
 
   // Package vendor
   vendor string
+
+  // Package installation path
+  installLocation string
```

Comment:
Can we not use files() for this?

Reply:
I first thought so too, but files is about the files which are inside the package and extracted to disk. E.g. for a dpkg package, it is about the files stored in `/usr`, the man pages, ...

This is about the installer itself, as I understood the Windows data.

Reply:
I think we should keep this consistent with macOS:

```
packages.list[0]: {
  name: "App Store"
  epoch: ""
  purl: "pkg:macos/macos/App%20Store@3.0?arch=arm64"
  arch: "arm64"
  outdated: false
  files: [
    0: pkgFileInfo path="/System/Applications/App Store.app"
  ]
  cpes: []
  format: "macos"
  version: "3.0"
  available: ""
  description: ""
  installed: true
  origin: ""
  vendor: ""
  status: ""
}
```